test/unit_tests/apps/teachers-digital-platform/js/survey/grade-level-page-spec.js

```diff
@@ -1,36 +1,50 @@
 import { jest } from '@jest/globals';
-import surveys from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js';
-import Cookies from 'js-cookie';
 import {
   ANSWERS_SESS_KEY,
   RESULT_COOKIE,
   SURVEY_COOKIE,
 } from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/config.js';
-import * as modals from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js';
 
 const HTML_SNIPPET = `
 <div data-tdp-page="grade-level">
 </div>
 `;
 
-xdescribe('The TDP survey grade-level page', () => {
-  beforeEach(() => {
+describe('The TDP survey grade-level page', () => {
+  let surveys;
+  const remove = jest.fn();
+  const init = jest.fn();
+
+  beforeAll(async () => {
+    jest.unstable_mockModule('js-cookie', () => ({
+      default: {
+        remove,
+      },
+    }));
+
+    jest.unstable_mockModule(
+      '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js',
+      () => ({
+        init,
+        close: jest.fn(),
+      })
+    );
+
+    surveys = await import(
+      '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js'
+    );
+
     document.body.innerHTML = HTML_SNIPPET;
   });
 
   it('should be recognized from HTML', () => {
-    const cookieSpy = jest.spyOn(Cookies, 'remove');
-    const modalSpy = jest.spyOn(modals, 'init');
     sessionStorage.setItem(ANSWERS_SESS_KEY, 'testItem');
 
-    surveys.init();
+    surveys.default.init();
```

Comment:
Maybe at the point of import have:

```
surveysModule = await import(
      '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js'
    );
surveys = surveysModule.default;
```

That way we only need to touch the point of import when the import behavior changes in the future.

Reply:
changed in 7e7c611